src/frontends/paddle/docs/tests.md

```diff
@@ -0,0 +1,17 @@
+# OpenVINO Paddle Frontend Tests
+
+OpenVINO Paddle Frontend [tests](../tests) have the next structure:
+ * `test_models/gen_scripts/` - Python script to generate PaddlePaddle test models with handy helpers.
+ * `standalone_build` - PaddlePaddle frontend standalone build test.
+ * a couple of files - PaddlePaddle frontend tests suite written using Google Test.
```

Comment:
can we be more specific about this couple of files or is it enough info for the users?

Reply:
updated